easybuild/easyconfigs/n/NAG/NAG-26-intel-2018a.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'Binary'
+
+name = 'NAG'
+version = '26'
+
+homepage = 'http://www.nag.co.uk'
+description = """The worlds largest collection of robust, documented, tested and maintained numerical algorithms."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
```

Comment:
@SimonPinches Hmm, it doesn't make much sense to provide easyconfigs for `NAG` with both the `GCC` and `intel` toolchains I think...

If you want a `NAG` installation that is compatible with both `foss` and `intel` toolchains, just add a single easyconfig that uses `GCCcore` as a toolchain.

There's probably no need to include `binutils` as build dep in this case, unless the `install.sh` script actually compiles stuff.

Reply:
I understand your point but the libraries really are different (see https://www.nag.co.uk/content/downloads-nag-fortran-library-versions#linux).  I will nevertheless, change the `<GCC>` to be `<GCCcore>`.

Reply:
Travis wasn't happy when using the `<GCCcore>` toolchain without a `<binutils>` build dependency.